ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -31,6 +31,7 @@ sunbird_encryption_key={{sunbird_encryption_key}}
 sunbird_encryption_mode={{sunbird_encryption_mode}}
 sunbird_account_name={{sunbird_public_storage_account_name}}
 sunbird_account_key={{sunbird_public_storage_account_key}}
+sunbird_account_endpoint={{sunbird_public_storage_account_endpoint}}
```

Comment:
@ddevadat can we assign value to this variable as {{cloud_public_storage_endpoint}} ?

Reply:
we can make it generic like cloud_public_storage_endpoint, but as the other variables were of other format
sunbird_public_storage_account_name
sunbird_public_storage_account_key

So we gave a name like sunbird_public_storage_account_endpoint

Reply:
Lets have other variables also changed as per how it is mentioned in the private repo template so that adopter need not add additional mapping in common.yaml like sunbird_public_storage_account_name: "{{ cloud_public_storage_accountname }}"
Below are the variable names mentioned in the private repo template
cloud_public_storage_accountname
cloud_public_storage_endpoint
cloud_public_storage_secret

Reply:
updated as per generic names

Reply:
@ddevadat changes are not reflecting. Can you please check and update?

Reply:
i added it yesterday
https://github.com/dicdiksha/sunbird-devops/commit/944c16ef8ab27f9ab6b61cbf9d83d59d6a3d64b6